Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.cs

```diff
@@ -34,11 +34,15 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     [ContentProperty(Name = "Content")]
     public partial class Expander : ContentControl
     {
+        /// <summary>
+        /// Default constructor for the Expander control
```

Comment:
The proper constructor wording is `Initializes a new instance of the <see cref="Expander" /> class` @